src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -929,8 +929,12 @@ internal static Cci.IGenericParameterReference Translate(TypeParameterSymbol par
             DiagnosticBag diagnostics,
             bool needDeclaration = false)
         {
+            if (fieldSymbol.IsTupleField)
```

Comment:
It is not required at the moment, I believe, but this is a correct way to deal with them in emit.

---
In reply to: [61794411](https://github.com/dotnet/roslyn/pull/11006#discussion_r61794411) [](ancestors = 61794411)